frontend/src/app/patients/EditPatient.test.tsx

```diff
@@ -347,7 +360,8 @@ describe("EditPatient", () => {
   describe("facility select input", () => {
     let component: any;
     beforeEach(async () => {
-      jest.useFakeTimers().setSystemTime(new Date("2021-08-01").getTime());
+      MockDate.set("2021-08-01");
+      //jest.useFakeTimers().setSystemTime(new Date("2021-08-01").getTime());
```

Comment:
nit: we can remove this commented out line :D

Reply:
thanks for catching this!